### PR TITLE
Convex_hull_3: Fix constness

### PR DESCRIPTION
**Pull request by afabri**
##  Summary of Changes

Add `const`  in the signature of several functions.  The problem did not show up in the examples and testsuite as the polyhedron passed to these functions is not const.      It shows up in the example put on [gist.gitub.com](https://gist.github.com/afabri/ec9cb00c0c321d91389649b858576782)

This PR should replace PR #5146

## Release Management

* Affected package(s): Convex_hull_3

### CODE REVIEW

File: Convex_hull_3/include/CGAL/convexity_check_3.h [hunk 1]
```diff
@@ -187,13 +187,13 @@ bool is_strongly_convex_3(const Polyhedron& P, const Traits& traits)
...
+bool CGAL_is_strongly_convex_3(const Polyhedron& P, Point_3<R>*)
 {
   return is_strongly_convex_3(P, R());
 }
 
 template<class Polyhedron>
-bool is_strongly_convex_3(Polyhedron& P)
+bool is_strongly_convex_3(const Polyhedron& P)
 {
   typedef typename boost::property_map<Polyhedron, vertex_point_t>::type Ppmap;
```
Comment (sloriot) [inline Convex_hull_3/include/CGAL/convexity_check_3.h:198]:
```suggestion
  typedef typename boost::property_map<Polyhedron, vertex_point_t>::const_type Ppmap;
```

File: Convex_hull_3/examples/Convex_hull_3/CMakeLists.txt [hunk 2]
```diff
@@ -42,6 +42,8 @@ endif()
 # Creating entries for all C++ files with "main" routine
 # ##########################################################
 
+create_single_source_cgal_program( "is_strongly_convex_3.cpp" )
```
Comment (sloriot) [inline Convex_hull_3/examples/Convex_hull_3/CMakeLists.txt:45]: you did not commit the test

Comment (afabri) [reply Convex_hull_3/examples/Convex_hull_3/CMakeLists.txt:45]: Let me remove it again.  In fact it shows that our testsuite globally does not address these constness issues.

File: Convex_hull_3/include/CGAL/convexity_check_3.h (hunk 1 above)
Comment (afabri) [reply Convex_hull_3/include/CGAL/convexity_check_3.h]: I changed it, but note that testsuite and examples passed locally without the change.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.2-Ic-50.shtml

**Pull request merged**